### PR TITLE
Feature/me does

### PR DESCRIPTION
**Pull request by Sudrien**
XEP-0245 for this one

### CODE REVIEW

File: me-does/candy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var CandyShop = (function(self) { return self; }(CandyShop || {}));
+
+CandyShop.MeDoes = (function(self, Candy, $) {
+
+	self.init = function() {
+
+		//$(Candy).on("candy:core.chat.message", function(e, args) {
```
Comment (mweibel) [inline me-does/candy.js]: Could you remove this commented out line?

File: me-does/candy.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		$(Candy).on("candy:view.message.before-show", function(e, args) {
+			if (args && args.message && args.message.match(/^\/me /i)) {
+                                var message = args.message.match(/^\/([^\s]+)(?:\s+(.*))?$/m)[2];
```
Comment (mweibel) [inline me-does/candy.js]: Wrong indent here (one space too much)

**Comment by mweibel** [comment]
Thanks for the PR. Will merge as soon as I'm happy with the code style.

**Comment by Sudrien** [comment]
It's great how PuTTY copies tabs as spaces. Should be ready now.

**Comment by Sudrien** [comment]
Side note: I will have a PR for the slash-commands plugin that allows /me to pass through, but I'm looking to implement /invite and /kick before doing so.

**Comment by melissanoelle** [comment]
I think I've got some changes to the slash-commands plugin, but some of them are specific to the client code base I have it integrated in. I'll pick out the non-Candy specific stuff and update it... if I'm remembering correctly I think it may be helpful to you.

**Pull request merged**